packages/lavapack/src/runtime.js

```diff
@@ -93,13 +93,19 @@
     } = __lavamoatSecurityOptions__
 
     // identify the globalRef
-    const globalRef = (typeof globalThis !== 'undefined') ? globalThis : (typeof self !== 'undefined') ? self : (typeof global !== 'undefined') ? global : undefined
+    let globalRef = globalThis
+
     if (!globalRef) {
-      throw new Error('Lavamoat - unable to identify globalRef')
+      globalRef = self || global
```

Comment:
referencing undeclared vars can lead to failures in strict mode

Reply:
we already had "unsafe" access to `self` and `global` before this change #459